src/server/routes/api/index.js

```diff
@@ -18,36 +17,94 @@ function getSqsPayload(object) {
   };
 }
 
+function validateSqsData(params, type) {
```

Comment:
@ktp242 -- updated, simplified the logic checks and returned a better response object to easily debug

Reply:
:+1